commands/svc.cmd

```diff
@@ -45,6 +45,13 @@ fi
 
 WARDEN_PHPMYADMIN_ENABLE="${WARDEN_PHPMYADMIN_ENABLE:-1}"
 if [[ "${WARDEN_PHPMYADMIN_ENABLE}" == 1 ]]; then
+    if [[ -d "${WARDEN_HOME_DIR}/etc/phpmyadmin/config.user.inc.php" ]]; then
+        rm -rf ${WARDEN_HOME_DIR}/etc/phpmyadmin/config.user.inc.php
+    fi
```

Comment:
I don't think you can remove this directory when the container is running.

Could you confirm that?

Since changes with phpmyadmin isn't added to release yet, I suggest not covering this case at all (remove these lines), but provide a manual instruction for those who affected.

Reply:
Just checked - on Mac it worked fine. If it also works fine for other OS, I don't mind keeping it as is

Reply:
This case works even when the whole .warden directory is deleted. 
I tested it when SVC containers ran: I removed the ".warden" directory and started SVCs again.

Running the command after the incorrect type of entity is created:
![image](https://github.com/user-attachments/assets/e8466338-7d0b-4921-a38f-8f94d450dd9b)

I propose to leave all the lines in the PR.